py/dml/dmlc.py

```diff
@@ -502,11 +508,13 @@ def show_illegal_attributes(option, opt, value, parser):
             else:
                 f = open(options.makedep, 'w')
             with f:
-                # rebuild from DML if imported file has changed
-                f.write('%s %s.c : %s\n' % (
-                    options.makedep, outputbase, deps))
-                # if imported file vanished, survive and rebuild
-                f.write('%s :\n' % (deps,))
+                    # rebuild from DML if imported file has changed
```

Comment:
This line looks wrong w.r.t number of spaces. I think python will choke on this. I expect old line 505 and new line 511 to be identical.

Reply:
Indeed, this is something for @mandolaerik to dig into

Reply:
Python ignores the indentation of all blank lines, with or without comments.